content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/create-github-issues/index.md

```diff
@@ -0,0 +1,235 @@
+
+1. `group`: your Chainguard group ID. Use `chainctl iam groups describe <your group> -o json |jq -r '.id'` to retrieve it.
+
+2. `project_id`: your Google Cloud project ID. Use the human readable name for the project as opposed to the numeric identifier.
+
+3. `github_org`: a Github organization or username that hosts the repository where you will create issues.
+
+4. `github_repo`: the Github repository inside the organization or user's account where you will create issues.
+
+5. `labels`: a comma separated list of labels that you would like to apply to the generated Github issues.
+
+Once you have all the information gathered, run the `terraform plan` command:
+
+```shell
+terraform plan \
+  -var "github_org=<user or org>" \
+  -var "github_repo=<repo name>" \
+  -var "group=<chainguard group ID>" \
+  -var "project_id=<gcloud project ID>" \
+  -var "name=enforce-events" \
+  -var "labels=enforce-events" \
+  -out enforce-events.plan
+```
+
+You will receive output showing Terraform's execution plan like the following:
+
+```
+. . .
+# google_cloud_run_service.gh-iss will be created
+  + resource "google_cloud_run_service" "gh-iss" {
+      + autogenerate_revision_name = false
+      + id                         = (known after apply)
+      + location                   = "<google location>"
+      + name                       = "enforce-events-issue-opener"
+      + project                    = "<google project>"
+      + status                     = (known after apply)
+. . .
+To perform exactly these actions, run the following command to apply:
+    terraform apply "enforce-events.plan"
+```
+
+Inspect the plan to ensure that the resources that Terraform will create are in line with your Google Cloud project's settings and IAM permissions. Once you are satisfied with the plan, apply it:
+
+```shell
+terraform apply enforce-events.plan
+```
+
+Terraform will run the module and you will receive output like the following:
+
+```
+google_secret_manager_secret.gh-pat: Creating...
+google_service_account.gh-iss-opener: Creating...
+. . .
+Outputs:
+
+secret-command = "echo -n YOUR GITHUB PAT | gcloud --project <google project> secrets versions add enforce-events-github-pat --data-file=-"
+url = "https://enforce-events-issue-opener-ntfj7ezpja-uc.a.run.app"
+```
+
+Note down the `url`, and the secret-command.
+
+## Configuring Google Cloud with your Github Token
+
+With the output from the `terraform apply` step, you can now run the `echo` command to configure Google Cloud with your Github personal access token.
+
+Substitute your Github personal access token in place of the `YOUR GITHUB PAT` string and run the following command:
+
+```shell
+echo -n <your github PAT> | gcloud --project <google project> secrets versions add enforce-events-github-pat --data-file=-
+```
+
+This configures your Google Cloud project with your token, so that it is available to the demo application when it is invoked on Cloud Run.
```

Comment:
```suggestion
This configures your Google Cloud project with your token so that it is available to the demo application when it is invoked on Cloud Run.
```